src/module/actor/influence/sheet.ts

```diff
@@ -0,0 +1,87 @@
+import { CreaturePF2e } from "@actor";
+import { InfluenceSheetData } from "@actor/influence/types";
+import { InfluenceDifficultyClass } from "@actor/creature/data";
+
+class InfluenceSheetPF2e<TActor extends CreaturePF2e> extends ActorSheet<TActor> {
+    static override get defaultOptions() {
+        const options = super.defaultOptions;
+        return foundry.utils.mergeObject(options, {
+            classes: [...options.classes, "pf2e", "influence"],
+            template: "systems/pf2e/templates/actors/influence/sheet.hbs",
+        });
+    }
+
+    override get title(): string {
+        return game.i18n.format("PF2E.InfluenceSheet.Title", { name: this.actor.name });
+    }
+
+    override async getData(): Promise<InfluenceSheetData<TActor>> {
+        const sheetData = super.getData() as InfluenceSheetData<TActor>;
+
+        sheetData.editable = false;
+        sheetData.label = {
+            rarity: CONFIG.PF2E.rarityTraits,
+            size: CONFIG.PF2E.actorSizes,
+            statistic: {
+                perception: "PF2E.PerceptionLabel",
+                ...CONFIG.PF2E.skillList,
+            },
+            trait: CONFIG.PF2E.monsterTraits,
+        };
+
+        return sheetData;
+    }
+
+    override activateListeners($html: JQuery): void {
+        super.activateListeners($html);
+        const html = $html[0];
+
+        html.querySelectorAll("section[data-statistic]:not([data-statistic=''])").forEach((section) => {
```

Comment:
use `htmlQueryAll` so as to not have to check whether it's an `HTMLElement`

Reply:
Ah, sweet!